src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.cs

```diff
@@ -712,7 +712,7 @@ [.. variableInfos.Select(v => Argument(v.ReturnBehavior == ReturnBehavior.Initia
                 }
             }
 
-            protected override async Task<GeneratedCode> CreateGeneratedCodeAsync(
```

Comment:
this type existed to hold onto a document and SyntaxAnnotations.  but the syntax annotations could be static-readonlys.  so no need for this wrapper.